src/Commands/UpdateIndexCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Rapidez\Core\Commands;
+
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Storage;
+use Rapidez\Core\Events\IndexAfterEvent;
+use Rapidez\Core\Events\IndexBeforeEvent;
+use Rapidez\Core\Facades\Rapidez;
+use Rapidez\Core\Models\Traits\Searchable;
+
+class UpdateIndexCommand extends Command
```

Comment:
I can imagine cases where this command takes longer than a minute to run, causing overlaps which can definitely be problematic.

We should make sure it can't overlap, either within the recommended scheduling usage (`->withoutOverlapping()`) or maybe within the command itself.

Reply:
I've updated the doc 👍 
It shouldn't cause issues since it will just override the already changed product information, but it wouldn't make sense to let that happen indeed.